app/models/user_manager.rb

```diff
@@ -44,6 +46,17 @@ def error_message
 
   private
 
+  def check_correct_admin_roles
+    admin_access = (admin.roles & REQUIRED_ADMIN_ROLES).any?
+
+    if admin_access
+      true
+    else
+      errors << "Must have authorized admin role to proceed"
```

Comment:
Now that I've seen this in action, I am not a fan. I would like it to be more like the message in https://github.com/redsummernight/otwarchive/pull/6/files#diff-55c5b7aecfb519d0e4880eaf2788eb6eR287, e.g. "Sorry, only an authorized admin can do that."